### PR TITLE
Update ChangeLog.md for PowerShell 6.0.0 GA release

### PR DESCRIPTION
**Pull request by daxian-dbw**
## PR Summary

Update ChangeLog.md for PowerShell 6.0.0 GA release

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [ ] Issue filed - Issue link:
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+## v6.0.0 - 2018-01-09
+
+### Breaking changes
+
+- Remove `sc` alias which conflicts with `sc.exe` (#5827)
+- Move group policy settings and enable policy controlled logging in PowerShell Core (#5791)
+
+### Engine updates and fixes
+
+- remoting: Handle `DLLImport` failure of `libpsrpclient` (#5622)
```
Comment (adityapatwardhan) [inline CHANGELOG.md]: Maybe capitalize `remoting -> Remoting`

**Review by adityapatwardhan — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: Updated. Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## v6.0.0 - 2018-01-09
+
+### Breaking changes
+
+- Remove `sc` alias which conflicts with `sc.exe` (#5827)
+- Move group policy settings and enable policy controlled logging in PowerShell Core (#5791)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Instead of `Move`, perhaps use `Separate`?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (daxian-dbw) [reply CHANGELOG.md]: That's better. I will update.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## v6.0.0 - 2018-01-09
```
Comment (TravisEz13) [inline CHANGELOG.md]: Are we releasing today or tomorrow?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+
+### Test
+
+- Replace `lee.io` Tests with `WebListener` (#5709) (Thanks @markekraus!)
+- Update the docker based release package tests (#5692)
+- Replace Remaining `HttpBin.org` Tests with `WebListener` (#5665) (Thanks @markekraus!)
+
+### Build and Packaging Improvements
+
+- Update `MSI` packages to not install the x86 package as an x64 component and to use different `UpgradeCode` between x86 and x64 packages (#5812) (Thanks @bergmeister!)
```
Comment (TravisEz13) [inline CHANGELOG.md]: Maybe `Update 'Msi' x86 and x64 packages not to overwrite each other`

**Review by anmenaga — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+- Separate group policy settings and enable policy controlled logging in PowerShell Core (#5791)
+
+### Engine updates and fixes
+
+- Handle `DLLImport` failure of `libpsrpclient` in PowerShell Remoting on Unix platforms (#5622)
+
+### Test
+
+- Replace `lee.io` Tests with `WebListener` (#5709) (Thanks @markekraus!)
+- Update the docker based release package tests (#5692)
```
Comment (TravisEz13) [inline CHANGELOG.md]: Maybe `Update the docker base release package tests due to Pester module removal and other issues`

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
I this the MSI description should be fixed.  the other comments are not blocking

**Comment by daxian-dbw** [comment]
@SteveL-MSFT @TravisEz13 Your comments have been addressed. Thanks!

**Review by TravisEz13 — APPROVED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by daxian-dbw** [comment]
After merging #5837, this PR shows "This branch is out-of-date with the base branch". I will do a rebase to resolve it.

**Review by TravisEz13 — APPROVED** [review]

**Comment by daxian-dbw** [comment]
The macOS build was not started in the failed Travis CI for some reason ☹️ Given that this is doc change only and the spelling check was successful in the Linux build of the same CI, I will not restart the macOS build.

**Pull request merged**